tox.ini

```diff
@@ -1,30 +0,0 @@
-[tox]
-skipsdist = true
-envlist = py27, flake8
-
-[testenv]
-passenv = BUILD_NUMBER DISPLAY GIT_BRANCH GIT_COMMIT GIT_URL JENKINS_URL \
-    JOB_NAME PYTEST_ADDOPTS PYTEST_BASE_URL SAUCELABS_USR \
-    SAUCELABS_PSW
-deps = -rrequirements/tests.txt
-commands = pytest \
-    --html=results/{envname}.html --self-contained-html \
-    --junit-xml=results/{envname}.xml \
-    --log-raw=results/{envname}_raw.txt \
-    --log-tbpl=results/{envname}_tbpl.txt \
-    {posargs}
-
-[testenv:flake8]
-skip_install = true
-deps = -rrequirements/flake8.txt
-commands = flake8 {posargs:.}
-
-[flake8]
-ignore = E501
-
-[pytest]
-addopts = --verbose -r=a --driver=Firefox
-testpaths = tests
-xfail_strict = true
-base_url = https://web-mozillians-staging.production.paas.mozilla.community
-sensitive_url = mozillians\.org
```

Comment:
This section needs to be moved to `setup.cfg` or `pytest.ini`

Reply:
Done in https://github.com/mozilla/mozillians-tests/pull/400/commits/3728005a64b1d756b30f727d80bdaecd2d6aa5ce